C/common/include/asset_tracking.h

```diff
@@ -0,0 +1,71 @@
+#ifndef _ASSET_TRACKING_H
+#define _ASSET_TRACKING_H
+/*
+ * FogLAMP asset tracking related
+ *
+ * Copyright (c) 2018 OSisoft, LLC
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Amandeep Singh Arora
+ */
+#include <logger.h>
+#include <vector>
+#include <sstream>
+#include <unordered_set>
+
+class AssetTrackingTuple {
```

Comment:
Please add doxgen compatible comments to classes and methods.

Reply:
I added a comment at the top : "The AssetTrackingTuple class is used to represent an asset tracking tuple. Hash function and == operator are defined for this class and pointer to this class that would be required to create an unordered_set of this class." That covers the complete details.